src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -138,6 +143,10 @@ export default {
 		realCategoryIds() {
 			return _without(this.categoryIds, ...customCategoryIds);
 		},
+		getImageEnhancementExperimentVersion() {
```

Comment:
You don't need all these getters, you can just use `:image-enhancement-experiment-version="imageEnhancementExperimentVersion"` in the template